compiler-plugin/src/main/kotlin/com/google/devtools/ksp/processing/impl/ResolverImpl.kt

```diff
@@ -352,7 +352,11 @@ class ResolverImpl(
     internal fun mapToJvmSignatureInternal(declaration: KSDeclaration): String? = when (declaration) {
         is KSClassDeclaration -> resolveClassDeclaration(declaration)?.let { typeMapper.mapType(it).descriptor }
         is KSFunctionDeclaration -> resolveFunctionDeclaration(declaration)?.let {
-            typeMapper.mapAsmMethod(it).descriptor
```

Comment:
This just caught my eyes. Note that, as per https://youtrack.jetbrains.com/issue/KT-48532, old backend will be deprecated, and so will be `KotlinTypeMapper` (separately discussed at https://youtrack.jetbrains.com/issue/KT-48773/Investigate-the-possibility-of-removing-dependency-on-old-JVM-ba). You can also refer to http://b/186742973 or https://github.com/JetBrains/intellij-community/commit/4f41d9888c5b9f71d3e8b7864879eb7eec8704d4 to see how FE1.0 UAST avoids using it (well, replicate it w/ simpler/necessary part of generic rewriting).

Reply:
Thanks for the heads up, follow up in #1006